### PR TITLE
PANW Cortex - support XDR Analytics alerts 

### PR DESCRIPTION
**Pull request by Itay4**
## Status
Ready

## Description
Francesco Vigo added XDR Analytics alerts support for fetch incidents.

## Required version of Demisto
4.1.0

## Does it break backward compatibility?
   - No

## Must have
- [x] Tests
- [x] Documentation
- [x] Code Review

## Dependencies
None

### CODE REVIEW

File: Integrations/PaloAltoNetworksCortex/PaloAltoNetworksCortex.yml [hunk 1]
```diff
@@ -89,6 +90,57 @@ configuration:
...
+  defaultvalue: ""
+  type: 16
+  required: false
+  options:
+  - all
+  - Info
+  - Low
+  - Medium
+  - High
+- display: Category of alerts to fetch (XDR)
```
Comment (glicht) [inline Integrations/PaloAltoNetworksCortex/PaloAltoNetworksCortex.yml]:
```suggestion
- display: Category of alerts to fetch (XDR Analytics)
```

File: Integrations/PaloAltoNetworksCortex/PaloAltoNetworksCortex.yml [hunk 2]
```diff
@@ -89,6 +90,57 @@ configuration:
   - wildfire-virus
   required: false
   type: 16
+- display: Severity of alerts to fetch (XDR)
```
Comment (glicht) [inline Integrations/PaloAltoNetworksCortex/PaloAltoNetworksCortex.yml]:
```suggestion
- display: Severity of alerts to fetch (XDR Analytics)
```

File: Integrations/PaloAltoNetworksCortex/PaloAltoNetworksCortex.yml [hunk 3]
```diff
@@ -89,6 +90,57 @@ configuration:
...
+  name: xdr_severity
+  defaultvalue: ""
+  type: 16
+  required: false
+  options:
+  - all
+  - Info
+  - Low
+  - Medium
+  - High
```
Comment (glicht) [inline Integrations/PaloAltoNetworksCortex/PaloAltoNetworksCortex.yml]: Reverse the order here. High at the top as we do with the other severity drop downs.

File: Integrations/PaloAltoNetworksCortex/PaloAltoNetworksCortex.yml [hunk 4]
```diff
@@ -89,6 +90,57 @@ configuration:
...
+  - New Administrative Behavior
+  - Remote Command Execution
+  - Reverse Connection
+  - SMB/KRB Traffic from Non-Standard Process
+  - wmiprsve.exe Rare Child Process
+  - wsmprovhost.exe Rare Child Process
+  - Large Upload (FTP)
+  - Large Upload (Generic)
+  - Large Upload (HTTPS)
+  - Large Upload (SMTP)
```
Comment (glicht) [inline Integrations/PaloAltoNetworksCortex/PaloAltoNetworksCortex.yml]: Let's order this list according to alphabet

**Comment by NeelRust** [comment]
@fvigo can you push the incident layout as well?

**Pull request merged**